cmd/nerdctl/run_test.go

```diff
@@ -449,3 +449,76 @@ COPY --from=builder /go/src/logger/logger /
 	assert.Check(t, strings.Contains(log, "foo"))
 	assert.Check(t, strings.Contains(log, "bar"))
 }
+
+// this container only works properly with -t -d (or -t) modes
+const TerminalContainerDockerfile = `
+FROM golang:latest as builder
```

Comment:
Please pin the golang version and go.sum.

Also, can we avoid using golang here?

Reply:
> Please pin the golang version and go.sum.
> 
> Also, can we avoid using golang here?


We can package the container in advance and just provide the container name, what do you think?

Reply:
I'd prefer to minimize dependency on third party images unless it is unavoidable

Reply:
I thought of an easier way, just run the container directly (bash or /bin/sh), this requires -t to run properly
No need to build image anymore